lib/mongo/cluster.rb

```diff
@@ -442,6 +442,21 @@ def addresses
       addresses_list
     end
 
+    # The logical session timeout value in minutes.
+    #
+    # @example Get the logical session timeout in minutes.
+    #   cluster.logical_session_timeout
+    #
+    # @return [ Integer, nil ] The logical session timeout.
+    #
+    # @since 2.5.0
+    def logical_session_timeout
+      servers.inject(nil) do |min, server|
+        break unless timeout = server.logical_session_timeout
```

Comment:
This returns nil if any server's timeout is nil?

Reply:
Yes. The ``break`` ends the iteration returning nil from the block.

Reply:
Great, looks good to me.

Reply:
thanks